pkg/statesolver/buildkit.go

```diff
@@ -122,7 +122,25 @@ func (s BuildkitSolver) FileExists(
 	filepath string,
 	source *builddef.Context,
 ) (bool, error) {
-	_, err := s.ReadFile(ctx, filepath, s.FromContext(source))
+	// @TODO: switch to StatFile once it got fixed upstream (it currently fails the hard way if the given filepath does not exist).
```

Comment:
maybe you have a link to this issue to add to this comment ?

Reply:
Not yet, this bug hasn't been reported so far but I discussed about it with Buildkit maintainers :wink: